doc/loggers.md

```diff
@@ -563,4 +564,18 @@ redispub:
   text-format: ""
   buffer-size: 100
   redis-channel: dns-collector
+```
+
+### Falco
+
+Falco plugin Logger
```

Comment:
Perhaps it should be nice to have the link to the other plugin on falcon side ?

Reply:
Updated.